aries-site/src/examples/components/tag/CreateTag/CreateTag.js

```diff
@@ -0,0 +1,213 @@
+import React, { useContext, useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Form,
+  FormField,
+  Select,
+  Tag,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { ThemeContext } from 'styled-components';
+import { tagData } from '../data';
+import {
+  alphabetize,
+  AppContainer,
+  getRegExp,
+  TagPageHeader,
+  prefix,
+  TagResults,
+  updateCreateOption,
+} from '.';
+
+const defaultNameOptions = [];
+const defaultValueOptions = [];
+const defaultName = '';
+const defaultValue = '';
+const defaultTags = [];
+
+tagData.forEach(datum => {
+  const [name] = Object.keys(datum);
+  const [values] = Object.values(datum);
+  defaultNameOptions.push({
+    label: name,
+    values,
+  });
+});
+
+export const CreateTag = () => {
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+
+  // name
+  const [nameOptions, setNameOptions] = useState(defaultNameOptions);
+  const [name, setName] = useState(defaultName);
+  const [nameSearch, setNameSearch] = useState('');
+
+  // value
+  const [valueOptions, setValueOptions] = useState(defaultValueOptions);
+  const [value, setValue] = useState(defaultValue);
+  const [valueSearch, setValueSearch] = useState('');
+
+  // tags
+  const [currentTags, setCurrentTags] = useState(defaultTags);
+
+  useEffect(() => {
+    const nextValueOptions = name
+      ? defaultNameOptions.filter(n => n.label === name)[0].values
+      : defaultValueOptions;
+    setValueOptions(nextValueOptions);
+  }, [name]);
+
+  return (
+    <AppContainer>
+      <TagPageHeader />
+      <Form
+        messages={{ required: 'This is a required field.' }}
+        onSubmit={() => {
+          const exists = currentTags.filter(
+            t => t.name === name && t.value === value,
+          );
+          if (!exists.length) setCurrentTags([...currentTags, { name, value }]);
+          setName(defaultName);
+          setValue(defaultValue);
+        }}
+      >
+        <Grid
+          columns={
+            size !== 'small'
+              ? [['auto', 'medium'], ['auto', 'medium'], 'auto']
```

Comment:
I'm losing our responsive grid here, but I can't spot what's going on.
![Screen Shot 2021-12-21 at 9 59 35 AM](https://user-images.githubusercontent.com/1756948/146969480-70afa0cc-db22-458f-bb71-a4810cc00d9f.png)

Reply:
Is this before we hit small breakpoint -- or you've hit it now?

Reply:
Already hit the breakpoint.